src/simulation/SimulationManager.java

```diff
@@ -452,6 +471,12 @@ private boolean initializeWithModel(File model) {
             }
         }
 
+        int transRelIndex = modelString.indexOf(String.format("pred %s", getParsingConf().getTransitionRelationName()));
```

Comment:
Do we need to do this check for the `stateSigName` as well since we are already doing it for the `transitionRelationName` and `initPredicateName`.

Reply:
No, that error would be caught earlier as a syntax error if the sig name used in the `init` and/or `next` pred is not defined.